src/apps/pillarx-app/components/PortofolioOverview/PortofolioOverview.tsx

```diff
@@ -0,0 +1,101 @@
+import { useTranslation } from 'react-i18next'
+
+// types
+import { ApiData } from '../../../../types/api'
+
+// images
+import DefaultLogo from '../../images/logo-unknown.png'
+import BlendIcon from '../../images/blend-icon.svg'
+import RoutingIcon from '../../images/routing-icon.svg'
+
+// components
+import Tags from '../Tags/Tags'
+import TileContainer from '../TileContainer/TileContainer'
+import Body from '../Typography/Body'
+import H1 from '../Typography/H1'
+import WalletAddressOverview from '../WalletAdddressOverview/WalletAddressOverview'
+import TokensList from '../TokensList/TokensList'
+import TokensPercentage from '../TokensPercentage/TokensPercentage'
+import SkeletonLoader from '../../../../components/SkeletonLoader'
+
+
+type PortofolioOverviewProps = {
+    data: ApiData | undefined;
+    isDataLoading: boolean;
+}
+
+
+const PortofolioOverview = ({ data, isDataLoading }: PortofolioOverviewProps) => {
+    const [t] = useTranslation();
+    const { assets = [], total_pnl_history = {}, total_wallet_balance } = data || {};
```

Comment:
Do you think we should define a default value here for `total_wallet_balance`?

Reply:
sorry what do you mean?